### PR TITLE
feat!: remove duktape as it has no long been supported

### PR DESCRIPTION
**Pull request by LycrusHamster**
**What type of PR is this?**

feat

**What this PR does / why we need it**:

feat!: remove duktape as it has no long been supported

### CODE REVIEW

**Comment by yejiayu** [comment]
@LycrusHamster re-rebase

File: services/riscv/src/vm/syscall/debug.rs [hunk 1]
```diff
@@ -25,7 +25,7 @@ impl<Mac: ckb_vm::SupportMachine> ckb_vm::Syscalls<Mac> for SyscallDebug {
 
         let msg = get_str(machine, ptr)?;
         log::debug!(target: "riscv_debug", "{}", msg);
-
+        println!("{}", msg);
```
Comment (yejiayu) [inline services/riscv/src/vm/syscall/debug.rs]: delete

File: services/riscv/src/tests/general.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        pvm_tx_nonce(nonce);
+        pvm_ret_str((char*)nonce);
+
+    }else{
+        pvm_ret_str("not match test case");
+        return 69;
+    }
+
+    return 0;
+}
```
Comment (yejiayu) [inline services/riscv/src/tests/general.c]: new line

File: Makefile [hunk 3]
```diff
@@ -126,15 +126,15 @@ contract_test: libpvm.a
...
-pvm_docker:
-	$(DOCKER_BUILD) "cd /src && make pvm_structs_test"
+libpvm.a_docker:
+	$(DOCKER_BUILD) "cd /src && make libpvm.a"
+
+pvm_general_test: libpvm.a
+	$(CC) -I$(RISCV_SRC) $(TEST_SRC)/general.c $(RISCV_SRC)/libpvm.a $(LDFLAGS) -o $(TEST_SRC)/general
+
+pvm_general_test_docker:
+	$(DOCKER_BUILD) "cd /src && make pvm_general_test"
```
Comment (yejiayu) [inline Makefile]: new line

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (yejiayu) [inline yarn.lock]: delete yarn.lock from root dir

Comment (LycrusHamster) [reply yarn.lock]: oooops, it seems I wanna type yarn install somewhere else, but lose focus on another terminal.

**Pull request merged**